scripts/license-checker.mjs

```diff
@@ -0,0 +1,55 @@
+import licenseChecker from 'license-checker';
+import { resolve } from 'node:path';
+
+// valid excluded licenses
+const EXCLUDED_LICENSES = [
+  'MIT',
+  'ISC',
+  'Apache-2.0',
+  'BSD-3-Clause',
+  'BSD-2-Clause',
+  'BlueOak-1.0.0',
+  'CC0-1.0',
+  '0BSD',
+  'CC-BY-4.0',
+  'MIT*',
+  'WTFPL',
+  'MIT-0',
+  'Python-2.0',
+  'Public Domain',
+  'CC-BY-3.0',
+  'BSD*',
+  'UNLICENSED',
```

Comment:
@alvarosabu are we sure `UNLICENSED` would be a valid one? From a quick search, it seems that it might be the equivalent to explictly define it as copyrighted, no granting permissions.

(To clarify, I mean only `UNLICENSED`. When it comes to `Unlicense`, that one seems to be a valid open source one https://opensource.org/license/unlicense)

Reply:
You are right @alexjoverm I just shared this with the team on the Slack thread. I will remove it if we all agree